src/content/docs/infrastructure/amazon-integrations/manage-aws-data/aws-integrations-metrics.mdx

```diff
@@ -4574,6 +4595,162 @@ We no longer add these mappings, and we're committed to preserving all the new m
+    1. <DNT>**Fill data gaps with**</DNT>. Leave this void, or use Last known value if gaps in the data coming from AWS lead to false positives or negatives.
+
+See our documentation on [how to create NRQL alerts](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/) for more details.
+
+## Tags collection [#tags-collection]
+
+New Relic provides enhanced dimensions from metrics coming from AWS CloudWatch metric streams. Resource and custom tags are automatically pulled from most services and are used to decorate metrics with additional dimensions. Use [metrics and events](/docs/query-your-data/explore-query-data/explore-data/introduction-data-explorer/) to see which tags are available on each AWS metric.
+
+The following query shows an example of tags being collected and queried as dimensions in metrics:
+
+```
+SELECT average(`aws.rds.CPUUtilization`) FROM Metric FACET `tags.mycustomtag` SINCE 30 MINUTES AGO TIMESERIES
+```
+
+Not all metrics have their custom tags as dimensions. Only metrics linked to entities listed in the [entity explorer](/docs/new-relic-one/use-new-relic-one/ui-data/new-relic-one-entity-explorer) have their custom tags associated. The AWS CloudWatch metric stream doesn't include tags as part of the stream message; hence, additional processing is required on the New Relic side.
+
+## Metadata collection [#metadata-collection]
+
+Like with custom tags, New Relic also pulls metadata information from relevant AWS services in order to decorate AWS CloudWatch metrics with enriched metadata collected from AWS Services APIs. This metadata is accessible in New Relic as additional dimensions on the metrics provided by AWS CloudWatch.
+
+<Callout variant="important">
+  This optional capability is complementary to the CloudWatch Metric Streams integration. Your service discovery and monitoring will be incomplete if you don't enable this optional capability.
+</Callout>
+
+The solution relies on [AWS Config](https://docs.aws.amazon.com/config/index.html), which might incur in additional costs in your AWS account. AWS Config provides [granular controls](https://docs.aws.amazon.com/config/latest/developerguide/select-resources.html) to determine which services and resources are recorded. New Relic will only ingest metadata from the available resources in your AWS account.
+
+The following services and namespaces are supported:
+
+* ALB/NLB
+* API Gateway (excluding API v1)
+* DynamoDB
+* EBS
+* EC2
+* ECS
+* ELB
+* Lambda
+* RDS
+* S3
+
+### Infrastructure agent metrics and EC2 metadata decoration [#infra-data-decoration]
+
+As with the EC2 API polling integration, when the infrastructure agent is installed on a host and the EC2 namespace is active via AWS CloudWatch metric stream integration, then all the infrastructure agent events and metrics are decorated with additional metadata.
```

Comment:
Again, metric stream > Metric Stream

Reply:
Probably best to do a search across the doc for all instances just to double check